.gitignore

```diff
@@ -4,6 +4,8 @@
 BowPlayground
 bin
 sha_*.txt
+.travis.yml
```

Comment:
I wouldn't include the `.travis.yml` here. Just removing it from the repo should be enough.